Formula/drill.rb

```diff
@@ -0,0 +1,35 @@
+class Drill < Formula
+  desc "HTTP load testing application written in Rust"
+  homepage "https://github.com/fcsonline/drill"
+  url "https://github.com/fcsonline/drill/archive/0.7.0.tar.gz"
+  sha256 "0041e4b8e8bae0bfc4dfc42b0d1c9a65a8a04dfb68766138d3ae85407e42b15f"
+  license "GPL-3.0-or-later"
+
+  depends_on "rust" => :build
```

Comment:
```suggestion
  depends_on "rust" => :build
  depends_on "openssl@1.1"
```

Reply:
Isn't the macOS version really old and deprecated?

Reply:
I don't actually know if it uses the macOS one. I only added this because it seems to need OpenSSL on Linux.

I couldn't get it to link with OpenSSL on macOS, so it may be using some other crate.

Reply:
Might be better to use `on_linux` then, same in the install block

Reply:
I think we actually did this for many rust formulae, like:
- [ncspot](https://github.com/Homebrew/homebrew-core/pull/81884)
- [oha](https://github.com/Homebrew/homebrew-core/pull/82426)
- [trunk](https://github.com/Homebrew/homebrew-core/pull/75650)
- [volta](https://github.com/Homebrew/homebrew-core/pull/72933)

Reply:
I would say we can go for the merge and then do the batch updates for all the rust formulae which has `uses_from_macos "openssl@1.1"`

Reply:
any thoughts cc @SMillerDev @carlocab ?